vignettes/adtte.Rmd

```diff
@@ -205,6 +204,63 @@ We advise you consult the `{admiral}`
 ["Creating a BDS Time-to-Event ADaM" vignette](https://pharmaverse.github.io/admiral/articles/bds_tte.html)
 for further guidance on the different options available and more examples.
 
+One extra common oncology case we include here is around PFS when censoring
+at next anti-cancer therapy. This could either be controlled using `ANLzzFL` as
+explained in the ADRS vignette, so that records after next anti-cancer therapy
+never contribute to the PD and DEATH parameters. Or alternatively you can control
+this on the ADTTE side by filtering which records are used in
+`admiral::event_source()` and `admiral::censor_source()`.
+The censor could be set as whichever date your analysis requires,
+e.g. date of last tumor assessment prior to next anti-cancer therapy or last
+radiological assessment.
+This case is demonstrated in the below example (where `NACTDT` would be
+pre-derived as first date of next anti-cancer therapy and `LASTANDT` as date of
+last assessment prior to next anti-cancer therapy).
+
```

Comment:
if you have not any preference can we change "next anti-cancer therapy" in above paragraph to "new  anti-cancer therapy". Also, to add to the last sentence 'LASTANDT" as " as date of
last assessment prior to new anti-cancer therapy. **If no new anti-cancer therapy for the participant then this date variable will have his last disease assessment date**". 

Also, if can also add general statement " For PD or Death event date if we want to consider conditionally for time to event analysis like not want to consider after new anti-cancer therapy or 2 missed assessment than we should filter them first using event_source."
Also suggest, for censoring at different EVNTDESC values, we can create multiple censor_source conditions and pass to the censor_conditions. But please make sure censor source object condition should be true only once of the participant..

Reply:
Working through these updates and will make most, but i feel the general statement is already covered by the following and the example: 
"Or alternatively you can control this on the ADTTE side by filtering which records are used in `admiral::event_source()` and `admiral::censor_source()`."

Also for the last point it doesn't have to be specifically set-up this way, the user just needs to know that it takes the maximum of these.

Reply:
Thanks Ross, sorry I didn't find the information in event_source() (https://pharmaverse.github.io/admiral/reference/event_source.html),  but if its there and clear then its fine.

Also, for the second point, only starting max. doesn't helped as I tried in my testing. 2 points/reasons 1. It can give wrong results depending on how censor dates condition are provided. if a same patient has 2 dates for eg censor_source() objects provided from 2 different condition say for eg last assessment before new anti-cancer therapy and last assessment before PD, in these 2 cases we need to select minimum, but we decided not to adjust for this in our programming. So I have to only pass one date per patient( except TRTSDT/RANNDT) , if I have created multiple censor_source objects.  2. Now why I want to send multiple censor_source date objects ?- I want to different EVNTDESC/CNSTDESC values to be distinguish for different censoring rules.

I hope I am able to explain it well.

Reply:
gotcha @amitjaingsk i added some more commits and hopefully i now captured your comments. please take another look